addons/website_event/tests/test_ui.py

```diff
@@ -1,11 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-import odoo.tests
-from odoo import tools
-
-
-@odoo.tests.tagged('post_install', '-at_install')
-class TestUi(odoo.tests.HttpCase):
```

Comment:
Why removing this test ?

Reply:
As far as I understand, this test only ran the tour that I removed, so of course, it failed after I did so. And it did not seem useful to keep a test that do nothing. I might be wrong !

Reply:
Well we should keep the tour as a classic test but not as a user tour. Keeping tests is always a good idea :D .